regtests/ww3_ufs_weather/input/rt.conf.wave

```diff
@@ -0,0 +1,19 @@
+###################################################################################################################################################################################
+# CPLD tests                                                                                                                                                                      #
```

Comment:
This is a duplicate of what's in ufs-weather-model and if we need a rt.conf.wave, I think it should live in ufs-weather-model not ww3